### PR TITLE
Backlog: omit metadata/log folders

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: src/dashboard/src/components/ingest/views.py [hunk 1]
```diff
@@ -520,20 +520,23 @@ def transfer_backlog(request, ui):
...
+        query = elasticSearchFunctions.MATCH_ALL_QUERY.copy()
+        query['filter'] = elasticSearchFunctions.LOG_FILE_FILTER
     else:
         queries, ops, fields, types = advanced_search.search_parameter_prep(request)
 
         try:
+            # Specify this as a filter, not a must_have, for performance,
+            # and so that it doesn't cause the "should" queries in a
+            # should-only query to be ignored.
+            filter = {'term': {'status': 'backlog'}}
```
Comment (Hwesta) [inline src/dashboard/src/components/ingest/views.py]: `filter` is a builtin...

**Comment by Hwesta** [comment]
:+1:

File: src/dashboard/src/components/ingest/views.py (hunk 1 above)
Comment (mistydemeo) [inline src/dashboard/src/components/ingest/views.py]: arg, right. I'll rename this "filters"

**Comment by Hwesta** [comment]
Still :+1:

**Pull request merged**